Update apple-mdm-push-certificate-get.md (#3079)
MicrosoftDocs/memdocs: memdocs/intune/enrollment/apple-mdm-push-certificate-get.md

> [!NOTE]
> The certificate is associated with the Apple ID used to create it. As a best practice, use a company email address as your Apple ID and make sure the mailbox is monitored by more than one person, such as by a distribution list. Avoid using a personal Apple ID.  

> 
> > [!NOTE]
> > We saw that alternate browser (e.g. brave) don't download a .pem file but a .cer. Using Microsoft Edge or Safari as a browser allow to have the proper file.

Reviewer: ```suggestion
>
> We have noticed that alternate browsers, such as Brave, don't download a .pem file but a .cer file. Using Microsoft Edge or Safari as a browser allows you to download the correct file.
```
